tests/scripts/task_kg_test.sh

```diff
@@ -51,9 +52,10 @@ if [ "$2" == "cpu" ]; then
 
     # verify saving training result
     dglke_eval --model_name DistMult --dataset FB15k --hidden_dim 100 \
-        --gamma 500.0 --batch_size 16 --eval_percent 0.01 --model_path ckpts/DistMult_FB15k_0/ \
```

Comment:
do we no longer create a directory for a model? the model is by default saved in ckpts?

Reply:
I think we still need it. The action of creating the folder is done in get_logger which is deleted. I will find a place in the trainer to create the folder